proto/cosmos/crypto/keyring/types.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package cosmos.crypto.keyring;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "cosmos/crypto/hd/hd.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keyring";
+option (gogoproto.goproto_getters_all) = false;
+
+message Record {
+  string name = 1;
+  google.protobuf.Any pub_key = 2;
+
+  oneof item {
+    Local local = 3;
+    Ledger ledger  = 4;
+    Empty empty  = 5;
+  }
+
+  // Item is a keyring item stored in a keyring backend.
+  message Local {
+    string privKeyArmor = 1;
```

Comment:
Can we use `Any priv_key = 1;` here? In that way, we defer encryption to 99designs, and we don't manage any password.

Reply:
Yes  - same as here: https://github.com/cosmos/cosmos-sdk/pull/9222#discussion_r624685677